pro/src/design-system/SnackBar/SnackBar.tsx

```diff
@@ -0,0 +1,176 @@
+import cx from 'classnames'
+import fullClearIcon from 'icons/full-clear.svg'
+import fullCloseIcon from 'icons/full-close.svg'
+import fullValidateIcon from 'icons/full-validate.svg'
+import { useCallback, useEffect, useRef, useState } from 'react'
+
+import { SvgIcon } from '@/ui-kit/SvgIcon/SvgIcon'
+
+import styles from './SnackBar.module.scss'
+
+const SHORT_TEXT_DURATION = 5000
+const LONG_TEXT_DURATION = 10000
+const SHORT_TEXT_THRESHOLD = 120
+
+export enum SnackBarVariant {
+  SUCCESS = 'success',
+  ERROR = 'error',
+}
+
+export type SnackBarProps = {
+  /**
+   * The variant of the snack bar
+   */
+  variant?: SnackBarVariant
+  /**
+   * The text of the snack bar
+   */
+  text: string
+  /**
+   * The callback function to be called when the snack bar is closed
+   */
+  onClose?: () => void
+  /**
+   * The test id of the snack bar
+   */
+  testId?: string
+  /**
+   * Whether the snack bar should automatically close or not
+   */
+  autoClose?: boolean
+}
+
+const ANIMATION_DURATION = 300 // should have the same value as `&.show transition` in `SnackBar.module.scss`
```

Comment:
Est-ce qu'on pourrait exporter cette const afin de l'utiliser dans les specs ? (cf. commentaires plus haut)
```suggestion
export const ANIMATION_DURATION = 300 // should have the same value as `&.show transition` in `SnackBar.module.scss`
```